analytics/analytics-ui/src/app/services/analytic-filter.pipe.ts

```diff
@@ -0,0 +1,18 @@
+import { Pipe, PipeTransform } from '@angular/core';
```

Comment:
This filter should not be located in the services directory as it is only used in the analytics component and therefore shouldn't be imported into every component.